earth2mip/diagnostic/utils.py

```diff
@@ -0,0 +1,47 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES.
+# SPDX-FileCopyrightText: All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import torch
+
+
+def filter_channels(
```

Comment:
We previously called a similar function geometry.sel_channel.  xarray uses the term `sel`. Let's not use the name `filter`.

Also, it is not specific to diagnostics so does not belong scoped in the `diagnostic` package. I also think `utils` is a bit of a red flag suggesting this function doesn't have a good home.

I still think we should delete this function. and there are many places in the code that use .index directly. If you want to add a function for this, then please refactor some of those places to use this :)

Reply:
do w/e you want. probably not in diagnosqtic subpackge since this is more general.